beetsplug/lastgenre/genres-tree.yaml

```diff
@@ -156,6 +156,31 @@
     - timba
     - twoubadou
     - zouk
+- classical:
+    - ballet
+    - baroque:
+        - baroque music
+    - cantata
+    - chamber music:
+        - string quartet
+    - classic
```

Comment:
Well, as it is now, `classic` is given as a subgenre of `classical`, so it might be okay. I just think it's a too vague, non-specific term to be useful as a genre tag.